public_html/templates/about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Thorne&Spindle:About
+		</title>
+		<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Playfair+Display" />
+		<style>
+		#title {
+			text-decoration: underline;
+		}
+		#description {
+			text-indent: 50px;
+		}
+		body {
+			font-family: Playfair Display;
+		}
+		.avatar {
+			width: 96px;
+			height: 96px;
+		}
+		ul {
+			list-style: none;
+			margin-right: 40px;
+		}
+		.rightBio {
+			text-align: right;
+		}
+		</style>
+	</head>
+	
+	<body>
+
+		<div id="title">Company
+		</div>
```

Comment:
if its small just open and close on same line

example 
`<div> Company</div>`